SDDC_FX3/Application.h

```diff
@@ -65,14 +68,17 @@
 #define INFINITE_TRANSFER_SIZE			 (0)
 
 #define APP_THREADS						 (1)
-// void null_func(uint8_t, ...)  // redefine DebugPrint if required 
-#define DebugPrint CyU3PDebugPrint
-
+// void null_func(uint8_t, ...)  // redefine DebugPrint if required
 extern void DebugPrint2USB ( uint8_t priority, char *msg, ...);
-#define DebugUSB DebugPrint2USB
+
+#ifndef _DEBUG_USB_  // #include "../Interface.h"
+#define DebugPrint (CyU3PDebugPrint)
+#else
+#define DebugPrint (DebugPrint2USB)
+#endif
 
 // detect pin for HF103
-#define LED_KIT		(54) // This is also UART_CTS
+#define LED_KIT		   (54)		// This is also UART_CTS
```

Comment:
unrelated change

Reply:
Here may be  I got problems with the tab in different editors. 
I want to change the way in witch BBRF103 is detected and not to use LED_KIT anymore...